Models/User.php

```diff
@@ -0,0 +1,69 @@
+<?php
+    namespace Models;
+
+
+    class User
+    {
+        private $name;
+        private $lastName;
+        private $userName;
+        private $password;
+        private $email;
+        
+        public  function getName()
+        {
+            return $this->name;
+        }
```

Comment:
Putting only "getName" seems a bit ambiguous. Think it's better to put getFirstName, what do you think?